app/bundles/CoreBundle/Helper/BuilderTokenHelper.php

```diff
@@ -61,17 +87,23 @@ public function __construct(MauticFactory $factory, $modelName, $viewPermissionB
      * @param string              $valueColumn The column that houses the value
      * @param CompositeExpression $expr        Use $factory->getDatabase()->getExpressionBuilder()->andX()
      *
+     * @throws \BadMethodCallException
+     *
      * @return array|void
      */
     public function getTokens(
-        $tokenRegex,
-        $filter = '',
-        $labelColumn = 'name',
-        $valueColumn = 'id',
+        string $tokenRegex,
+        string $filter = '',
+        string $labelColumn = 'name',
+        string $valueColumn = 'id',
         CompositeExpression $expr = null
     ) {
+        if (!$this->isConfigured) {
+            throw new \BadMethodCallException('You must call the "'.get_class($this).'"::configure method of this class first.');
```

Comment:
```suggestion
            throw new \BadMethodCallException('You must call the "'.get_class($this).'::configure()" method first.');
```

Reply:
I thought I'll be able to easily commit the change from the Github UI but it doesn't let me